rotkehlchen/tests/unit/test_zerionsdk.py

```diff
@@ -50,6 +73,33 @@ def test_query_all_protocol_balances_for_account(
     assert len(warnings) == 0
 
 
+# currently only tests pcdai
+@pytest.mark.parametrize('custom_ethereum_tokens', [INITIAL_TOKENS])
+def test_query_pooltogether(
+        ethereum_manager,
+        function_scope_messages_aggregator,
+        inquirer,  # pylint: disable=unused-argument
+        ethereum_manager_connect_at_start,
+):
+    wait_until_all_nodes_connected(
+        ethereum_manager_connect_at_start=ethereum_manager_connect_at_start,
+        ethereum=ethereum_manager,
+    )
+    zerion = ZerionSDK(ethereum_manager, function_scope_messages_aggregator)
+
+    balances = zerion.all_balances_for_account('0x12329cE52a04feA0908399b151CD0edBdd8237D2')
+    if len(balances) == 0:
+        test_warnings.warn(UserWarning('Test acc for Pooltogether (DAI) Compound has no balances'))
+        return
+    print(balances)
+    assert len(balances) > 0
+    assert balances[0].base_balance.balance.amount == 125500
```

Comment:
I'm not sure about this statement since the account  could move the tokens and make the test fail. Maybe @LefterisJP can make a better recommendation here?